rts/motoko-rts/Cargo.toml

```diff
@@ -36,7 +36,9 @@ enhanced_orthogonal_persistence = ["incremental_gc"]
 
 [dependencies]
 libc = { version = "0.2.139", default-features = false, optional = true }
+libm = "=0.2.7"
```

Comment:
This pins it. Done by `cargo add libm@=0.2.7`, because we only have a few versions available, following the vendoring of the crates that the nightly pulls in.